internal/resources/schemas/lb_profile_schemas.go

```diff
@@ -0,0 +1,426 @@
+package schemas
+
+import (
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/resources/validations"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func HttpProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "HTTP Profile configuration",
+		MaxItems:    1,
+		ExactlyOneOf: []string{"http_profile", "tcp_profile", "udp_profile",
+			"cookie_profile", "sourceip_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		ConflictsWith: []string{"tcp_profile", "udp_profile",
+			"cookie_profile", "sourceip_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"service_type": {
+					Type:             schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{"LBHttpProfile", "LBFastTcpProfile", "LBFastUdpProfile", "LBClientSslProfile", "LBServerSslProfile", "LBCookiePersistenceProfile", "LBGenericPersistenceProfile", "LBSourceIpPersistenceProfile"}, false),
+					Required:         true,
+					Description:      "Network Loadbalancer Supported values are `LBHttpProfile`,`LBFastTcpProfile`, `LBFastUdpProfile`, `LBClientSslProfile`,`LBServerSslProfile`, `LBCookiePersistenceProfile`,`LBGenericPersistenceProfile`,`LBSourceIpPersistenceProfile`",
+				},
+				"http_idle_timeout": {
+					Type:     schema.TypeInt,
+					Required: true,
+					//Default:          15,
```

Comment:
does it mean, there are no default for these fields ?

Reply:
we need these default for particular profile types removed the comment